lib/ts-morph.d.ts

```diff
@@ -6453,7 +6457,7 @@ export declare class JsxText extends JsxTextBase<ts.JsxText> {
     /**
      * Gets if the JSX text contains only white spaces.
      */
-    containsOnlyTriviaWhiteSpaces(): boolean;
+    containsOnlyTriviaWhiteSpaces(): any;
```

Comment:
Hmmm... I wonder why this happened. Did you use `yarn install` to install the dependencies or `npm install`?